ehr2meds/generate_synthetic_raw_data.py

```
@@ -0,0 +1,166 @@
+        missing_linked_cols = [col for col in linked_on if col not in linked_df.columns]
+        if missing_linked_cols:
+            raise ValueError(
+                f"Linked columns {missing_linked_cols} not found in linked file '{linked_file}'. "
+                f"Available columns: {list(linked_df.columns)}"
+            )
+
+        # Get columns
+        linked_cols = linked_df[linked_on]
+
+        if linked_type == "choice":
+            selected_idx = random.randint(0, len(linked_cols) - 1)
```

Ruff (0.15.10)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)